spec/support/vcr_cassettes/mr_client/bb_internal/apigw_get_radiology.yml

```diff
@@ -0,0 +1,104 @@
+---
+http_interactions:
+- request:
+    method: get
+    uri: "<MHV_MR_HOST>/v1/bluebutton/radiology/phrList/11382904"
+    body:
+      encoding: US-ASCII
+      string: ''
+    headers:
+      Accept:
+      - application/json
+      Content-Type:
+      - application/json
+      User-Agent:
+      - Vets.gov Agent
+      Token: "<SESSION_TOKEN>"
+      X-Api-Key:
+      - "<MHV_MR_X_API_KEY>"
+      Accept-Encoding:
+      - gzip;q=1.0,deflate;q=0.6,identity;q=0.3
+  response:
+    status:
+      code: 200
+      message: OK
+    headers:
+      Date:
+      - Tue, 25 Mar 2025 14:22:51 GMT
+      Content-Type:
+      - application/json
+      Content-Length:
+      - '6866'
+      X-Amzn-Requestid:
+      - 9ab93464-9872-4a62-8a3f-786574ad5afa
+      X-Amzn-Remapped-Connection:
+      - keep-alive
+      X-Amz-Apigw-Id:
+      - H_JZWG_cPHMFTFQ=
+      X-Amzn-Remapped-Date:
+      - Tue, 25 Mar 2025 14:22:51 GMT
+      Strict-Transport-Security:
+      - max-age=16000000; includeSubDomains; preload;
+    body:
+      encoding: ASCII-8BIT
```

Comment:
These are test user responses, correct? Just want to make sure we're not surfacingany PII